datasets/titanic/metadata.json

```diff
@@ -1,30 +1,70 @@
 {
     "@context": {
-      "@vocab": "https://schema.org/",
-      "csvw": "http://www.w3.org/ns/csvw",
-      "ml": "http://mlcommons.org/schema/"
+      "sc": "https://schema.org/",
+      "ml": "http://mlcommons.org/schema/",
+      "wd": "https://www.wikidata.org/wiki/Q"
     },
-  
-    "@type": "Dataset",
+
+    "@type": "sc:Dataset",
     "@language": "en",
     "name": "Titanic",
     "url": "https://www.openml.org/d/40945",
     "description": "The original Titanic dataset, describing the status of individual passengers on the Titanic.\n\n The titanic data does not contain information from the crew, but it does contain actual ages of half of the passengers. \n\n For more information about how this dataset was constructed: \nhttps://web.archive.org/web/20200802155940/http://biostat.mc.vanderbilt.edu/wiki/pub/Main/DataSets/titanic3info.txt\n\nOther useful information (useful for prices description for example):\nhttp://campus.lakeforest.edu/frank/FILES/MLFfiles/Bio150/Titanic/TitanicMETA.pdf\n\n Also see the following article describing shortcomings of the dataset data:\nhttps://emma-stiefel.medium.com/plugging-holes-in-kaggles-titanic-dataset-an-introduction-to-combining-datasets-with-fuzzywuzzy-60a686699da7\n",
     "license": "Public",
     "citation": "The principal source for data about Titanic passengers is the Encyclopedia Titanica (http://www.encyclopedia-titanica.org/). The datasets used here were begun by a variety of researchers. One of the original sources is Eaton & Haas (1994) Titanic: Triumph and Tragedy, Patrick Stephens Ltd, which includes a passenger list created by many researchers and edited by Michael A. Findlay.\n\nThomas Cason of UVa has greatly updated and improved the titanic data frame using the Encyclopedia Titanica and created the dataset here. Some duplicate passengers have been dropped, many errors corrected, many missing ages filled in, and new variables created.\n",
-  
+
     "distribution": [
       {
-        "@type": "FileObject",
+        "@type": "sc:FileObject",
         "name": "passengers-table",
         "contentUrl": "https://www.openml.org/data/get_csv/16826755/phpMYEkMl",
         "contentSize": "117743 B",
         "sha256": "c617db2c7470716250f6f001be51304c76bcc8815527ab8bae734bdca0735737",
         "encodingFormat": "text/csv"
       }
     ],
-  
+
     "recordSet": [
+      {
+        "@type": "ml:RecordSet",
+        "name": "gender_enums",
```

Comment:
Great question! I started out using @id, but then switched to name, because the semantics of id doesn't fully match what we're doing, as it's more about defining absolute URLs, and using fragments to abbreviate them rather than defining and referencing local identifiers. CSVW made the same choice.

We could revisit using @id if we think we can make it work. But things like referencing fields of RecordSets with a syntax like "#{rs1/field2}" are a bit of a stretch.